docs/tutorials/projects/make-a-plant-watering-robot.md

```diff
@@ -41,6 +41,7 @@ The tutorial uses the following hardware, but you can adjust it as needed:
 - A screwdriver
 
 Before starting this tutorial, follow the [Raspberry Pi Setup Guide](/installation/prepare/rpi-setup/) to prepare your Pi to run `viam-server`.
+Connect your Pi to its [power supply](https://www.amazon.com/CanaKit-Raspberry-Supply-Adapter-Listed/dp/B00MARDJZ4) to power it on.
```

Comment:
I'd unlink this. Users could provide any valid power supply -- not just this CanaKit one -- and we've already provided it as a recommendation above.

Reply:
Good point, done!